compiler/rustc_codegen_llvm/src/coverageinfo/mod.rs

```diff
@@ -17,14 +17,15 @@ use rustc_middle::mir::coverage::CoverageKind;
 use rustc_middle::mir::Coverage;
 use rustc_middle::ty::layout::HasTyCtxt;
 use rustc_middle::ty::Instance;
+use rustc_target::abi::Align;
 
 use std::cell::RefCell;
 
 pub(crate) mod ffi;
 pub(crate) mod map_data;
 pub mod mapgen;
 
-const VAR_ALIGN_BYTES: usize = 8;
+const VAR_ALIGN_BYTES: Align = Align::EIGHT;
```

Comment:
The variable name stops making sense if you change its type to `Align`. Probably rename this into `VAR_ALIGN`.